src/marks/difference.d.ts

```diff
@@ -69,6 +69,52 @@ export interface DifferenceOptions extends MarkOptions, CurveOptions {
   z?: ChannelValue;
 }
 
+export interface DifferenceXOptions extends DifferenceYOptions {
```

Comment:
We should make a shared `DifferenceOptions` based instead of having `DifferenceXOptions` extend `DifferenceYOptions`.

Reply:
Actually it seems to me that the types are identical, and only the comments are difference? So I’ll unify the interfaces.